how-to-capture-fiddler-trace.md

```diff
@@ -0,0 +1,37 @@
+---
+title: How to capture Fiddler trace
```

Comment:
Titles require title style capitalization. In this case, this would be "How to Capture Fiddler Trace"